momepy/functional/tests/test_dimension.py

```diff
@@ -84,3 +87,98 @@ def test_perimeter_wall(self):
 
         pd.testing.assert_series_equal(result, result_given_graph)
         assert result[0] == pytest.approx(137.210, rel=1e-3)
+
+    @pytest.mark.skipif(not GPD_013, reason="no attribute 'segmentize'")
+    def test_street_profile(self):
```

Comment:
Can we include this?
https://github.com/pysal/momepy/blob/1a89a9c0481e4498a0a385866779ef0fdd752600/momepy/tests/test_dimension.py#L361-L374